Configurateur/front/lib/components/custom_footer.dart

```diff
@@ -0,0 +1,149 @@
+                                25.0), // half of height and width of Image
+                          ),
+                          child: IconButton(
+                            tooltip: "Page de contact",
+                            icon: const Icon(
+                              Icons.contact_page,
+                              size: 20.0,
+                            ),
+                            color: Provider.of<ThemeService>(context).isDark
+                                ? darkTheme.primaryColor
+                                : lightTheme.primaryColor,
+                            onPressed: () {
+                              context.go("/contact");
+                            },
+                          ),
+                        ),
+                      )),
+                  SizedBox(
+                      height: 45.0,
+                      width: 45.0,
+                      child: Center(
+                        child: Card(
+                          elevation: 5.0,
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(
+                                25.0), // half of height and width of Image
+                          ),
+                          child: IconButton(
+                            tooltip: "Développé par RISU",
+                            icon: const Icon(
+                              Icons.fingerprint,
+                              size: 20.0,
+                            ),
+                            color: Provider.of<ThemeService>(context).isDark
+                                ? darkTheme.primaryColor
+                                : lightTheme.primaryColor,
+                            onPressed: () {},
+                          ),
+                        ),
+                      )),
+                  SizedBox(
+                      height: 45.0,
+                      width: 45.0,
+                      child: Center(
+                        child: Card(
+                          elevation: 5.0,
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(
+                                25.0), // half of height and width of Image
+                          ),
+                          child: IconButton(
+                            tooltip: "Page des avis",
+                            icon: const Icon(
+                              Icons.stars,
+                              size: 20.0,
+                            ),
+                            color: Provider.of<ThemeService>(context).isDark
+                                ? darkTheme.primaryColor
+                                : lightTheme.primaryColor,
+                            onPressed: () {
+                              context.go("/feedbacks");
+                            },
+                          ),
+                        ),
+                      )),
```

Comment:
Maybe add , after the first ) so that the file is more readable

Reply:
done